README.md

```diff
@@ -23,6 +23,9 @@ const getJwks = buildGetJwks({
   ttl: 60 * 1000,
   timeout: 5000,
   allowedDomains: ['https://example.com'],
+  checkIssuer: (domain) => {
+    return domain === 'https://example.com'
+  },
```

Comment:
let's be consistent with naming. if this function is called checkIssuer, the argument it takes should be something that identifies the issuer, not the domain